src/App.js

```diff
@@ -1,39 +1,64 @@
-import { useEffect, useState } from "react";
+import React, { useEffect, useState } from "react";
+import { Route, Redirect } from "react-router-dom";
 
 import UserProfile from "./components/User/UserProfile";
 import Photos from "./components/gallery/Photos";
+import Picture from "./components/Picture";
 import "./App.css";
 
 function App() {
-
   const [data, setData] = useState(null);
 
   useEffect(() => {
-     
-      let myHeaders = new Headers();
-      myHeaders.append("Origin", "https://localhost.com");
-      
-      let requestOptions = {
-        method: 'GET',
-        headers: myHeaders,
-        redirect: 'follow'
-      };
-
-      fetch("https://image-gallery-pjks.s3.ca-central-1.amazonaws.com/data.json", requestOptions)
-       .then(response => response.json())
-       .catch(() => console.log('add backup data'))
-       .then((response) => setData(response)); 
+    let myHeaders = new Headers();
+    myHeaders.append("Origin", "https://localhost.com");
```

Comment:
not sure if u read my previous comment
i told u to use window.origin instead of hardcoding it

Reply:
done